### PR TITLE
Replace `@` imports with `~` imports

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description

In the interest of consistency this PR replaces the few instances of `@` imports with `~`. Vue CLI apps generally tend to use `@` as the alias for the `src` dir, but Nuxt seems to [support both equally](https://nuxtjs.org/docs/configuration-glossary/configuration-alias/) so there's really no reason to favour one over the other. Since `~` is more commonly used in the code, using that as the standard causes minimum disruption.

## Testing Instructions

0. Check out the PR and run the dev server.
1. It should start.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by sarayourfriend** [comment]
> Vue CLI apps generally tend to use @ as the alias for the src dir

Would it actually be better for us to go with the upstream preference? It'd generally be nice if our app's configuration and behavior and expectations aligned with the wider Vue, and especially Vue 3, ecosystem.

File: src/components/FooterSection.vue [hunk 1]
```diff
@@ -67,8 +67,8 @@
 </template>
 
 <script>
```
Comment (obulat) [inline src/components/FooterSection.vue:69]: I wonder if we'll ever use this component...

**Review by krysal — APPROVED** [review]
Somehow I feel the `~` is more appropriate, maybe is a good distinction between a "Nuxt" project and just a "Vue" project? or I just get used to it. Anyway, LGTM.

**Comment by dhruvkb** [comment]
As long as we're internally consistent, I see no significant reason to pick one over the other. So I preferred the option with the least disruption.

**Comment by zackkrida** [comment]
I'm surprised there were so few uses of `@`. Nice cleanup

**Pull request merged**